bibliopixel/drivers/ledtype.py

```diff
@@ -0,0 +1,25 @@
+from enum import IntEnum
+
+
+class LEDTYPE(IntEnum):
+    GENERIC = 0  # Use if the serial device only supports one chipset
+    LPD8806 = 1
+    WS2801 = 2
+    # These are all the same
+    WS2811 = 3
```

Comment:
Trust me. This is never changing. To you and I, yes, it's illogical. But to our users it's necessary. For example... most users will call the WS2812 just "NeoPixel" which is just a Adafruit brand name for that chipset. But also there is the WS2812B chipset which is the same protocol, different form factor. Oh, and WS2811 which is the same exact protocol except not built into the LED but in an 8 pin SOP IC package. The problem here is that we've had users contact us and say "I have a WS2811 but you only support WS2812, can you please support that?". So this avoid problems. 
Another is APA102/APA102C/SK9822... it's all the same protocol despite different names. Again, we'd have lots of questions. It's just easier to add these duplicates in there because the users don't care what the protocol is, just that they see the name for the chipset they have.